### DISCUSSION TITLE
[A-L] (2023/24) Foglio 4 - Esercizio 14

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#302] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-11-02 185048](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/839bdae8-2f53-485c-813a-4a918f1a791c)

**Comment by Elia-Belli** [comment, marked as answer]
### Prodotto Cartesiano di Anelli
Un anello $(A,+,\cdot)$ soddisfa le seguenti proprietà:
1. $(A,+)$ è un gruppo commutativo;
2. L'operazione $\cdot$ è associativa;
3. Vale la distributività di $\cdot$ rispetto a $+$;
- **$(A \times B,+)$ è un gruppo commutativo**
Come mostrato in #51 il prodotto cartesiano di due gruppi commutativi ha naturale struttura di gruppo commutativo: per ipotesi $(A,+_A)$ e $(B,+_B)$ sono gruppi commutativi $\Rightarrow (A\times B,+)$ è un gruppo commutativo con elemento neutro $0:=(0_A,0_B)$

- **Associatività di $\cdot$**
Siano $a,a',a'' \in A, b,b',b'' \in B$, devo dimostrare che $((a,b)\cdot (a',b'))\cdot (a'',b'')=(a,b)\cdot ((a',b')\cdot (a'',b''))$:
$((a,b)\cdot (a',b'))\cdot (a'',b'')=(a\cdot_A a',b \cdot_B b')\cdot (a'',b'')=((a\cdot_A a') \cdot_A a'',(b \cdot_B b')\cdot_B b'')$
$(a,b)\cdot ((a',b')\cdot (a'',b''))=(a,b)\cdot(a'\cdot_A a'',b' \cdot_B b'')=(a\cdot_A (a' \cdot_A a''),b \cdot_B (b'\cdot_B b''))$
Che sono equivalenti per l'associatività di $\cdot_A,\cdot_B$

- **Distributività di $\cdot$ rispetto a $+$**
Siano $a,a',a'' \in A, b,b',b'' \in B$, devo dimostrare le seguenti due proprietà:
   1. $((a,b)+(a',b'))\cdot (a'',b'')=((a,b)\cdot (a'',b''))+((a',b')\cdot (a'',b''))$
   2. $(a'',b'')\cdot ((a,b)+ (a',b')) =((a'',b'')\cdot (a,b))+((a'',b'')\cdot (a',b'))$
Partiamo dalla (i):
$=(a+_Aa',b+_Bb')\cdot (a'',b'')=((a+_Aa')\cdot_A a'',(b+_Bb')\cdot_B b'')$, per distributività di $\cdot_A,\cdot_B$ rispetto a $+_A,+_B$ diventa:
$((a\cdot_Aa'')+_A(a'\cdot_Aa''),(b\cdot_Bb'')+_B(b'\cdot_Bb''))=((a\cdot_Aa''),(b\cdot_Bb''))+((a'\cdot_Aa''),(b'\cdot_Bb''))=$
$=((a,b)\cdot (a'',b''))+((a',b')\cdot (a'',b''))\ \square$
Analogamente (ii):
$=(a'',b'')\cdot((a+_Aa',b+_Bb'))=(a''\cdot_A(a+_Aa'),b''\cdot_B(b+_Bb'))$ per distributività di $\cdot_A,\cdot_B$ rispetto a $+_A,+_B$ diventa:
$((a''\cdot_Aa)+_A(a''\cdot_Aa'),(b''\cdot_Bb)+_B(b''\cdot_Bb'))=((a''\cdot_Aa),(b''\cdot_Bb))+((a''\cdot_Aa'),(b''\cdot_Bb'))=$
$=((a'',b'')\cdot (a,b))+((a'',b'')\cdot (a',b'))\ \square$

**Conclusione**: il prodotto cartesiano di anelli ha naturale struttura di anello.
### Prodotto Cartesiano di Anelli commutativi unitari
Per dimostrare che ciò vale anche per anelli commutativi unitari basta aggiungere le seguenti proprietà:
1. L'operazione $\cdot$ è commutativa
2. Esiste elemento neutro per $\cdot$

- **Commutatività di $\cdot$** :
$$(a,b)\cdot(a',b')=(a\cdot_A a',b\cdot_B b')=(a'\cdot_A a,b'\cdot_B b)=(a',b')\cdot(a,b)\ \cdot$$
- **Elemento neutro moltiplicativo**
Sia $1:=(1_A,1_B)$ l'elemento neutro commutativo:
$$(a,b)\cdot(1_A,1_B)=(a\cdot_A 1_A,b\cdot_B 1_B)=(a,b)\ \square$$
**Conclusione**: il prodotto cartesiano di anelli commutativi unitari ha naturale struttura di anello commutativo unitario.

**Comment by CiottoloMaggico** [comment]
Analogo @Elia-Belli 
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/5ff7946a-809b-4c68-a0f0-25003dd46370)
